# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/gc/GCEnum.h
# Commit: e590eb34efc5
# Full Hash: e590eb34efc52ab109f2737167cb2aa0975a1792
# Author: Iain Ireland <iireland@mozilla.com>
# Date: 2021-06-01 09:43:14
# Regressor Bug: 1554524
# File Overlap Count: 1
# Description:
#   Bug 1624792: Remove Debugger interface r=tcampbell
#   
#   This patch removes the changes from bug 1554524 part 4. (Parts 1 and 3 of bug 1554524 were bytecode emitter cleanups that don't need to be reverted; part 2 was abandoned.)
#   
#   Depends on D116308
# ==============================================================================

diff -r 135a0e6af641 -r e590eb34efc5 js/src/gc/GCEnum.h
--- a/js/src/gc/GCEnum.h	Tue Jun 01 01:31:21 2021 +0000
+++ b/js/src/gc/GCEnum.h	Tue Jun 01 01:31:21 2021 +0000
@@ -135,7 +135,6 @@
   _(DebuggerFrameIterData)                 \
   _(DebuggerOnStepHandler)                 \
   _(DebuggerOnPopHandler)                  \
-  _(RealmInstrumentation)                  \
   _(ICUObject)                             \
   _(FinalizationRegistryRecordVector)      \
   _(FinalizationRegistryRegistrations)     \